<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1233</width>
    <height>1627</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="sleep_button">
    <property name="geometry">
     <rect>
      <x>502</x>
      <y>910</y>
      <width>200</width>
      <height>200</height>
     </rect>
    </property>
    <property name="text">
     <string>睡觉</string>
    </property>
    <property name="iconSize">
     <size>
      <width>17</width>
      <height>17</height>
     </size>
    </property>
    <property name="tag" stdset="0">
     <number>0</number>
    </property>
   </widget>
   <widget class="QPushButton" name="eat_button">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>910</y>
      <width>200</width>
      <height>200</height>
     </rect>
    </property>
    <property name="text">
     <string>吃饭</string>
    </property>
    <property name="iconSize">
     <size>
      <width>17</width>
      <height>17</height>
     </size>
    </property>
    <property name="tag" stdset="0">
     <number>0</number>
    </property>
   </widget>
   <widget class="QPushButton" name="exercise_button">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>910</y>
      <width>200</width>
      <height>200</height>
     </rect>
    </property>
    <property name="text">
     <string>锻炼</string>
    </property>
    <property name="iconSize">
     <size>
      <width>17</width>
      <height>17</height>
     </size>
    </property>
    <property name="tag" stdset="0">
     <number>0</number>
    </property>
   </widget>
   <widget class="QPushButton" name="commute_button">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>1140</y>
      <width>200</width>
      <height>200</height>
     </rect>
    </property>
    <property name="text">
     <string>通勤</string>
    </property>
    <property name="iconSize">
     <size>
      <width>17</width>
      <height>17</height>
     </size>
    </property>
    <property name="tag" stdset="0">
     <number>0</number>
    </property>
   </widget>
   <widget class="QPushButton" name="break_button">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>1140</y>
      <width>200</width>
      <height>200</height>
     </rect>
    </property>
    <property name="text">
     <string>休息</string>
    </property>
    <property name="iconSize">
     <size>
      <width>17</width>
      <height>17</height>
     </size>
    </property>
    <property name="tag" stdset="0">
     <number>0</number>
    </property>
   </widget>
   <widget class="QPushButton" name="work_button">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>1140</y>
      <width>200</width>
      <height>200</height>
     </rect>
    </property>
    <property name="whatsThis">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="text">
     <string>工作</string>
    </property>
    <property name="iconSize">
     <size>
      <width>17</width>
      <height>17</height>
     </size>
    </property>
    <property name="tag" stdset="0">
     <number>0</number>
    </property>
   </widget>
   <widget class="QLabel" name="task_hint">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>260</y>
      <width>191</width>
      <height>141</height>
     </rect>
    </property>
    <property name="text">
     <string>当前任务</string>
    </property>
   </widget>
   <widget class="QLabel" name="current_task">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>250</y>
      <width>201</width>
      <height>141</height>
     </rect>
    </property>
    <property name="text">
     <string>未知任务</string>
    </property>
   </widget>
   <widget class="QLCDNumber" name="lcdNumber">
    <property name="geometry">
     <rect>
      <x>820</x>
      <y>260</y>
      <width>201</width>
      <height>121</height>
     </rect>
    </property>
    <property name="smallDecimalPoint">
     <bool>true</bool>
    </property>
    <property name="digitCount">
     <number>6</number>
    </property>
    <property name="mode">
     <enum>QLCDNumber::Dec</enum>
    </property>
    <property name="segmentStyle">
     <enum>QLCDNumber::Flat</enum>
    </property>
    <property name="value" stdset="0">
     <double>11.000000000000000</double>
    </property>
    <property name="intValue" stdset="0">
     <number>11</number>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>1410</y>
      <width>221</width>
      <height>171</height>
     </rect>
    </property>
    <property name="text">
     <string>上一页</string>
    </property>
    <property name="tag" stdset="0">
     <number>1</number>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>870</x>
      <y>1400</y>
      <width>241</width>
      <height>181</height>
     </rect>
    </property>
    <property name="text">
     <string>下一页</string>
    </property>
    <property name="tag" stdset="0">
     <number>1</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBox">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>1420</y>
      <width>321</width>
      <height>161</height>
     </rect>
    </property>
    <property name="maximum">
     <number>3</number>
    </property>
    <property name="value">
     <number>1</number>
    </property>
   </widget>
   <widget class="QTextBrowser" name="textBrowser">
    <property name="geometry">
     <rect>
      <x>195</x>
      <y>460</y>
      <width>751</width>
      <height>192</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
